### DISCUSSION TITLE
Unable to Enable JIT on iPadOS 17 Developer Beta 4

### DISCUSSION THREAD

**Discussion by ktalothegoat10** [SideStore/sidestore.github.io#27] [General]
I updated my iPad to iPadOS 17 Developer beta 4 and now whenever I want to enable JIT in SideStore I get an error that says "Unable to start Debug Server on this Device"

What version I'm using.
SideStore installed from Windows new Sideserver

Can Anyone please help, Thank you.

**Comment by ktalothegoat10** [comment]
https://github.com/SideStore/sidestore.github.io/assets/139594595/df6be1b2-1def-40d2-88a6-d30a739e1451

**Reply by ktalothegoat10** [reply]
Here is a video to show

**Comment by nythepegasus** [comment]
Hello! iOS 17 completely changed how JIT works. Also this is the wrong Github repository to report these types of issues. Please open an issue on the [SideStore](https://github.com/SideStore/SideStore/issues) repository instead.

**Comment by ktalothegoat10** [comment]
Ok thank you
